model_analyzer/config/input/config_command_profile.py

```diff
@@ -936,6 +938,24 @@ def _add_run_search_configs(self):
                 description="Maximum percentage of the search space to profile when using Optuna",
             )
         )
+        self._add_config(
+            ConfigField(
+                "optuna_min_trials",
```

Comment:
I see support for min trials, but I don't see it used anywhere?

Reply:
This is only relevant/used once the code for early exit is added. I've just added the option for it now.